pennylane/interfaces/torch.py

```diff
@@ -16,13 +16,69 @@
 compatible quantum nodes.
 """
 # pylint: disable=redefined-outer-name,arguments-differ
+from collections import Iterable
 import inspect
 from functools import partial
+import numbers
 
 import numpy as np
 import torch
+from torch.autograd.function import once_differentiable
 
-from pennylane.utils import unflatten
+
+def _unflatten_torch(flat, model):
+    """Restores an arbitrary nested structure to a flattened Torch tensor.
+
+    See also :func:`_flatten`.
+
+    Args:
+        flat (torch.Tensor): 1D array of items
+        model (array, Iterable, Number): model nested structure
+
+    Raises:
+        TypeError: if ``model`` contains an object of unsupported type
+
+    Returns:
```

Comment:
Does the order of raises and returns matter? It's different for the wrapper

Reply:
Oops! It shouldn't (Sphinx should extract both), but this should be fixed for consistency.